src/collections/resources/resources.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Given the data file for all resources is also named `resources.json`, I think it makes sense to rename either that file or this one in order to avoid any confusion. Perhaps the one that lives in `\dist` could be `resourceData.json`?